lib/active_admin/filters/humanized.rb

```diff
@@ -9,8 +9,22 @@ def initialize(param)
         @value = param[1]
       end
 
+      def related_class
+        @related_class ||=
+            begin
+              @body.split('_id').first.gsub(' ', '').classify.constantize
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.